README.md

```diff
@@ -52,22 +52,37 @@
 <br>
 
 
-## Instructions to run
+## Instructions to Install package
 
 * Pre-requisites:
-  - Python version > 3
+  - Python version >=3.4
   
 * Install the package using 
-```bash
-pip install pipgui==1.1
-```
+  ```bash
+  pip install pipgui==1.3
```

Comment:
Why should the users specify a version number?
they should just get the latest version

Reply:
okay ill remove that.

Reply:
removed version number from the readme